site/docs/external.md

```diff
@@ -307,7 +307,10 @@ bootstrap test.
 ### Repository rules
 
 Prefer [`http_archive`](repo/http.html#http_archive) to `git_repository` and
-`new_git_repository`.
+`new_git_repository`. Rationale: Git repository rules depend on system `git(1)` whereas
+the HTTP downloader is built into Bazel and has no system dependencies. `http_archive` 
+also supports a list of `urls` as mirrors, and `git_repository` supports only a single
+`remote`.
```

Comment:
Another reason to use `http_archive` is that it makes use of the repository cache: https://github.com/bazelbuild/bazel/issues/5116

Reply:
Thanks! Added.